src/composables/discover/useWorkflowTemplateSearch.ts

```
@@ -0,0 +1,100 @@
+import type { SearchResponse } from 'algoliasearch/dist/lite/browser'
+import { liteClient as algoliasearch } from 'algoliasearch/dist/lite/builds/browser'
```

ESLint

[error] Unable to resolve path to module 'algoliasearch/dist/lite/builds/browser'.

(import-x/no-unresolved)